Formula/rust-mode.rb

```diff
@@ -20,7 +20,7 @@ def caveats; <<-EOS.undent
     Add the following to your init file:
 
     (autoload 'rust-mode "rust-mode" nil t)
-    (add-to-list 'auto-mode-alist '("\\.rs\\'" . rust-mode))
+    (add-to-list 'auto-mode-alist '("\\\\.rs\\\\'" . rust-mode))
```

Comment:
<img src="https://cloud.githubusercontent.com/assets/2346707/16660954/fa9daa96-44b3-11e6-9ecf-cc998ba52056.jpg" width="128" />

Sometimes I can't believe I'm this funny.